src/ACEtk/interpretation/EL03/Bremsstrahlung.hpp

```diff
@@ -0,0 +1,42 @@
+class Bremsstrahlung {
+  const Table& table;
+
+
+public:
+  Bremsstrahlung( const Table& table ) : table( table ) {}
+  
+  auto electronEnergy() const {
+    const auto length = this->table.data.NXS( 5 );
+    const auto start  = this->table.data.JXS( 5 );
+    return this->table.data.XSS( start, length )
+      | ranges::view::transform( [](auto&& entry )
+				 { return entry * mega(electronVolt); } );
+  }
+
+  auto photonElectronEnergyRatio() const {
+    const auto length = this->table.data.NXS( 6 );
+    const auto start  = this->table.data.JXS( 5 ) + this->table.data.NXS( 5 );
+    return this->table.data.XSS( start, length );
+  }
+
+  auto values() const {
```

Comment:
It's not immediately obvious what bremsstrahlung values would refer to. I noticed there is no units attached. Is this multiplicity? Perhaps another name would be better.